src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
this propery doesn't inherit

Reply:
> Hi, I see you haven't fixed previous comments, maybe you did not push your changes? If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review 🙂

Yeah, I made a mistake and didnt push new files.
Now its okay.